components/LandingPage/LandingPage.tsx

```diff
@@ -112,14 +140,21 @@ const LandingPage = () => {
               profileImageRef={profileImageRef}
               openSoonImageRef={openSoonImageRef}
             />
+            <Footer />
           </div>
 
           <div className="absolute left-0 top-0 z-[3] w-[100%] h-[100%] pointer-events-none">
             <SectionContainer
               style={{
                 backgroundSize:
                   width > 985 ? `${width}px ${Number((width / 1439) * 975).toFixed(2)}px` : "985px",
-                height: width > 985 ? `${Number((width / 1439) * 975)}px` : "625px",
+                height:
+                  // eslint-disable-next-line no-nested-ternary
+                  width > 985
+                    ? `${Number((width / 1439) * 975)}px`
+                    : width > 390
+                    ? "625px"
+                    : "420px",
```

Comment:
same here, these breakpoints feel really arbitrary, can you assign them to `const` s so we know why you picked these, specific, `width` values?